src/components/Items/ProfileContact.js

```diff
@@ -61,7 +61,7 @@ const ProfileContact = ({
             <>
               {userId !== item.user_id && (
                 <View>
-                  <Mi name="arrow-forward-ios" size={18} color={COLORS.white2} />
+                  <Mi name="arrow-forward-ios" size={18} color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the arrow icon has been changed from `COLORS.white2` to `COLORS.white`. Make sure this color change doesn't affect the visibility of the icon against its background.
```diff
- <Mi name="arrow-forward-ios" size={18} color={COLORS.white2} />
+ <Mi name="arrow-forward-ios" size={18} color={COLORS.white} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->